src/command_modules/azure-cli-rdbms/azure/cli/command_modules/rdbms/help.py

```diff
@@ -0,0 +1,105 @@
+                type: group
+                short-summary: Commands to manage Azure %s server's firewall rules
+                """ % server_type
+    helps['{} server firewall-rule create'.format(commnad_group)] = """
+                type: command
+                short-summary: Creates a firewall rule
+                """
+    helps['{} server firewall-rule update'.format(commnad_group)] = """
+                type: command
+                short-summary: Updates a firewall rule
+                """
+    helps['{} server firewall-rule delete'.format(commnad_group)] = """
+                type: command
+                short-summary: Deletes a firewall rule
+                """
+    helps['{} server firewall-rule show'.format(commnad_group)] = """
+                type: command
+                short-summary: Show the details of a firewall rule
+                """
+    helps['{} server firewall-rule list'.format(commnad_group)] = """
+                type: command
+                short-summary: List all the firewall rules of a given server
+                """
+    helps['{} server configuration'.format(commnad_group)] = """
+                type: group
+                short-summary: Commands to manage %s server configurations
+                """ % server_type
+    helps['{} server configuration set'.format(commnad_group)] = """
+                type: command
+                short-summary: Updates the value of a server configuration
+                """
+    helps['{} server configuration show'.format(commnad_group)] = """
+                type: command
+                short-summary: Show the details of a server configuration
+                """
+    helps['{} server configuration list'.format(commnad_group)] = """
+                type: command
+                short-summary: List all the configurations of a given server
+                """
+    helps['{} server-logs'.format(commnad_group)] = """
+                type: group
+                short-summary: Commands to manage %s server logs.
+                """ % server_type
+    helps['{} db'.format(commnad_group)] = """
+                type: group
+                short-summary: Commands to manage %s databases
+                """ % server_type
+    helps['{} db create'.format(commnad_group)] = """
+                type: command
+                short-summary: Creates a database
+                """
+    helps['{} db delete'.format(commnad_group)] = """
+                type: command
+                short-summary: Deletes a database
+                """
+    helps['{} db show'.format(commnad_group)] = """
+                type: command
+                short-summary: Show the details of a database
+                """
+    helps['{} db list'.format(commnad_group)] = """
+                type: command
+                short-summary: List all the databases of a given server
+                """
+
+
+add_helps("mysql", "MySQL")
+add_helps("postgres", "PostgreSQL")
```

Comment:
I advise against using a loop in `help.py` file. The file is expected to be a fast access for documentation. Therefore a flat and clean format for faster accessibility to the command document trumps the need to reusability. Please flatten the loop.